### DISCUSSION TITLE
DCDNT / It works for a year+. Here's the recent Synthwave on it.

### DISCUSSION THREAD

**Discussion by mikepro4** [bangcorrupt/hacktribe#253] [Show and tell]
https://www.youtube.com/watch?v=lLrxnCh9mFw

Thanks to all who managed to do this firmware, this is literally the best thing that happened to my setup.

I wonder if we can upload custom wavetables on it somehow. Cheers all.

**Comment by mikepro4** [comment]
Also I'm curious. would you mind pointing me to to the way to change the logo on the starting screen. I like yours but it needs to be DCDNT in my case. Thanks.

**Reply by bangcorrupt** [reply]
This isn't an option yet, but I will add it as a feature in Hacktribe Editor some day.  You can get a greater say in the direction of Hacktribe by [becoming a sponsor](https://github.com/bangcorrupt/hacktribe/discussions/63).  I can't make any guarantees about what I'll work on or when it will be available, but I try to prioritise requests from people who support me.

**Reply by mikepro4** [reply]
@bangcorrupt thanks for all your work and so glad to see that this is still an ongoing project. I've signed up and would love to participate in this process.

**Reply by bangcorrupt** [reply]
Thank you for your support ! :smiley: 

Developing the editor is one of the most popular feature requests, once that's moved forward it will be easy to add an option to change the logo.

**Reply by bangcorrupt** [reply]
There is a [script](https://github.com/bangcorrupt/hacktribe/blob/main/scripts/ht_splash_screen.py) for replacing the boot image.  Bitmap should be 1 bit per pixel, 128x64, inverted monochrome.

The script checks that only the image bytes of the firmware have changed, so it should be safe.  If you want me to test with your custom logo, email it to me and I'll generate a hash of the patched firmware.

**Comment by AndrewSosna** [comment]
@bangcorrupt I have an idea, you could allow sponsors (maybe with specific sponsorship tier?) to set their own logo :)

**Reply by bangcorrupt** [reply]
I like the idea but I don't want to block features behind a paywall, even if they're only cosmetic.  I said at the start that everything will always be free and I think I should stick to that.  Nobody should have to pay anything, but everyone is welcome to pay as much they want.

**Comment by AndrewSosna** [comment]
@mikepro4 as for wavetables, as far as I remember you can just upload your single-shot samples  but not wavetables.
Nice song, btw

**Reply by mikepro4** [reply]
Thanks! But looks like @bangcorrupt figured out how to add additional wavetables as in FM modified ones. If this process is available for everybody we can use it like Serum.

**Reply by AndrewSosna** [reply]
I may be wrong, but I think he said that most (or some) oscillators have more parameters than exposed with physical knobs, so Korg just set some fixed value for one parameter and only exposed another parameter. So what @bangcorrupt did, if I'm not mistaken, is literally copying the same oscillator with different fixed values for that "hidden" parameter. I think he said that theoretically in future it may be possible to find a way to access those additional parameters maybe with Shift+Knob. At least that's what I remember from some other threads here.

**Reply by BKLronin** [reply]
Making a recorded sample really short like one period is essentially a custom wavetable that can be modified by start osc and decay poti. (Kind of)

**Reply by mikepro4** [reply]
So basically it's a single-cycle waveform approach. Is it actually currently possible?

**Reply by BKLronin** [reply]
Heaven't tried but the markers should be fine enough to hit the right spots. Loop point needs to be right too, might be difficult without visual feedback. 

Theoretically moving this window than gives you the different waveforms. Not very practical though.

**Reply by mikepro4** [reply]
@BKLronin so I wasn't sure whether sampling capabilities are still available after the hacktribe firmware. How do you actually try it? Is there a specific folder that samples can still go? Sorry I've never used anything related to sampling on Electribe.

**Reply by BKLronin** [reply]
Sampling is one of the main features, at least for me. They are on par with the sampler version. And then just using regular samples or even wavetables in wav format loaded the regular ways Id say. Have to try it myself.

**Reply by bangcorrupt** [reply]
> wavetables

There are no wavetables, @AndrewSosna  is correct about the oscillators.  At some point it may be possible to control the loop end point, allowing basic wavetables, but it's not an option yet.  

Hopefully I can get the Osc Edit drive fx from the synth working with custom samples.  I think it uses the same algorithms as the IFX, so most of the code is probably there.  

The KingKorg has an option to use 2 single cycle waveforms with phase modulation between them.   I'm not sure if there are any remnants of it in the electribe firmware, but it would be quite powerful with custom waveforms.

**Reply by BKLronin** [reply]
Just tried it, the loop controll points are fine enough the only thing is to hit a certain pitch. If the numbers in the sample edit are samples one might calculate how big the window must be or just use a guitar tuner. Once the pitch is locked you actually have a new static oscillator at least.

**Comment by enorrmann** [comment]
wavetables on the electribe would be amazing !
